### DISCUSSION TITLE
github api

### DISCUSSION THREAD

**Discussion by otaboyevsardorbek1** [community/community#64201] [Apps, API and Webhooks] [answered]
### Select Topic Area

Question

### _**using github api can i create new project and save it to server using python.!**_

**Comment by nirav-gajera** [comment]
Yesah, you can use the GitHub API to create a new project and save it to a server using Python.

Here are some steps :

1. Install the GitHub API library for Python. You can do this by running the following command in your terminal:

```
pip install github
```

2. Create a new Python script and import the GitHub API library.

```python
import github
```

3. Create a GitHub client object.

```python
client = github.Github(GITHUB_ACCESS_TOKEN)
```

4. Create a new repository object.

```python
repo = client.create_repo('my-new-project')
```

5. Save the repository to a server.

```python
repo.save()
```

Here is an example of a complete Python script that creates a new project and saves it to a server:

```python
import github

GITHUB_ACCESS_TOKEN = '<your-github-access-token>'

client = github.Github(GITHUB_ACCESS_TOKEN)

repo = client.create_repo('my-new-project')

repo.save()
```

You can replace `<your-github-access-token>` with your own GitHub access token. You can get your GitHub access token from the GitHub settings page: https://github.com/settings/tokens.

**Reply by otaboyevsardorbek1** [reply, marked as answer]
### _**thanks for the help**_
